php_phongo.c

```diff
@@ -2583,6 +2583,16 @@ PHP_MINFO_FUNCTION(mongodb)
 #else
 	/* Bundled libraries, buildtime = runtime */
 	php_info_print_table_row(2, "libmongoc version", MONGOC_VERSION_S);
+# if HAVE_MONGOC_SSL
```

Comment:
I'd suggest using the mongoc defines for this instead: MONGOC_ENABLE_SSL, MONGOC_ENABLE_SASL.
This should be updated later to print out with ssl lib it is using, maybe add a note to PHPC-542 ?